### DISCUSSION TITLE
Why is Upscayl so awesome!?

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#552] [Q&A]
Just how!?

**Comment by NayamAmarshe** [comment]
Because I made it.

**Reply by NayamAmarshe** [reply]
Thanks for the answer!

**Reply by NayamAmarshe** [reply]
You're welcome!

**Comment by aaronliu0130** [comment]
![Aaron Simpson backs into bushes](https://i.kym-cdn.com/entries/icons/original/000/015/976/homer-simpson-bush-gif.gif)

**Reply by chriswhite5918** [reply]
Is it possible to get the similar quality showcased in your website using Upscay? Turn that blurry landscape into a high quality photo, only with these upscaling models, how?

**Reply by aaronliu0130** [reply]
You... just Upscayl it?

**Reply by chriswhite5918** [reply]
I just want to know how to get the similar result showcased in your website.

**Reply by aaronliu0130** [reply]
Again, you just Upscayl it. I don't get what the problem you're encountering is.

**Reply by chriswhite5918** [reply]
<img width="744" alt="msedge_HBO7OzjICb" src="https://github.com/upscayl/upscayl/assets/36411355/50d355ab-5051-44a5-97ed-64526808e764">

I don't know how to get this using this tool, and I want to get such good results, any tutorial or guide?

**Reply by aaronliu0130** [reply]
You just run the app and follow its instructions.

**Reply by chriswhite5918** [reply]
Ok, I will check that, thanks.

**Reply by chriswhite5918** [reply]
I checked the instructions, and I just want to know if I can use this tool to make a blurry image clear? Like what is shown on the homepage of your website?

**Reply by aaronliu0130** [reply]
YEs.

**Reply by chriswhite5918** [reply]
That's fantastic! Could you write a guide for a similar use case? I couldn't find it on the wiki. I'm sure many users would find it useful, and this application could become very popular. A short instructional video would be sufficient as well.
